lib/src/message_formatter.dart

```diff
@@ -0,0 +1,28 @@
+import 'package:cmustify/src/metadata.dart';
+
+class MessageFormatter {
+  var metaData;
+
+  setMetaData(Metadata metaData) {
```

Comment:
No need for this. If you really want to add visibility, make `metadata`and private field (`_metadata`)  and use a setter `void set Metadata(Metadata metadata) { this.metadata = metadata; }`